privacyidea/lib/tokens/totptoken.py

```diff
@@ -60,6 +60,9 @@ class TotpTokenClass(HotpTokenClass):
     # but the last used OTP value, so we need to set this to 0.
     previous_otp_offset = 0
 
+    # The TOTP token provides means to verify the enrollment
+    can_verify_enrollment = True
```

Comment:
I think this is already inherited from the `HotpTokenClass`

Reply:
will remove it.

Reply:
Done in 8a660993